test/README.md

```diff
@@ -131,3 +132,29 @@ do day-to-day development as it allows you to get into a clean state with a sing
     ```
 
 7. Now you can run `dart test` and it should include the integration tests.
+
+## Manually configure Facebook, Google and Apple authentication providers
```

Comment:
Do we need these details in the Readme file. 
I think we better move much of these development details in `CONTRIBUTING.md`

Reply:
I suggeset we move the whole `Building the source` section to `contributing.md`

Reply:
and leave the section `Building the source` but link to the other file from it

Reply:
I made these changes in https://github.com/realm/realm-dart/pull/715